src/main/java/mate/academy/Main.java

```diff
@@ -44,7 +50,8 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
```

Comment:
In the mate/academy/Main.main() method create an instance of AuthenticationService using injector and test all methods from it.